tests/publiccloud/ssh_interactive_start.pm

```diff
@@ -16,17 +16,40 @@ use Mojo::Base 'publiccloud::ssh_interactive_init';
 use publiccloud::ssh_interactive;
 use testapi;
 use utils;
+use publiccloud::utils "select_host_console";
 
 sub run {
     my ($self, $args) = @_;
-    select_console 'tunnel-console';
+
+    # This ensure that we have used the setup console, even if no module was run before.
+    select_host_console();
+    my $setup_console = current_console();
 
     # Establish the tunnel (it will stay active in foreground and occupy this console!)
+    select_console('tunnel-console');
     ssh_interactive_tunnel($args->{my_instance});
 
-    # Switch to root-console and SSH to the instance
-    # every other loaded test must stay in root-console
-    select_console 'root-console';
+    # Enable ssh connection on setup console, this is done normally with the
+    # first activation hook in susedistribution:activate_console()
+    if ($setup_console !~ /tunnel/) {
+        select_console($setup_console);
+        script_run('ssh -t sut', timeout => 0);
+    }
```

Comment:
I didn't get this change.

Reply:
If we use the `tunnel` console, there is no need to do this `ssh -t sut` trick on the the setup_console. Cause it is already done by the fist activation of the console in this line: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/susedistribution.pm#L843

But if we use a different console (like the `root-virtio-terminal`) we need to setup this ssh connection later on.

Reply:
I still don't get it. We setup the tunnel by setting `TUNNELED=1` and calling `select_console 'root-console';`.